loader

```diff
@@ -144,7 +144,10 @@ def load(invocid, size, output_method, report_method, dist='gaussian', stddev=DE
     payloadgen = gen_methods[payload_gen]
     sep = ' - '
     sep_len = len(sep)
-    prefix = "loader seq - %s - " % invocid
+    if log_format == 'string':
```

Comment:
It would be better to use `if log_format_ == 'json':` for the json case, to make it explicit that the format is JSON

Reply:
+1
Fixing...